plugins/auth-backend/src/providers/auth0/provider.ts

```diff
@@ -61,20 +62,16 @@ export class Auth0AuthProvider implements OAuthHandlers {
         accessToken: any,
         refreshToken: any,
         params: any,
-        rawProfile: passport.Profile,
-        done: PassportDoneCallback<OAuthResponse, PrivateInfo>,
+        fullProfile: passport.Profile,
```

Comment:
Would it not make sense to keep calling these `rawProfile` (as above) or `passportProfile`? I thought the clarification was nice.

Reply:
I thought `rawProfile` didn't end up matching in the new context were it's actually being passed on to resolve the users. It's not `rawProfile -> profile` and then consume `profile` anymore, you end up needing both, with the `profile` just being the simplified version for the frontend settings page.

`passportProfile` could make sense too though, not sure if I think that's better than `fullProfile` though tbh.